Started Issue Tracker Home file with a table

gravinblumenn opened the pull request.

gravinblumenn committed:
Started Issue Tracker Home file with a table

gravinblumenn committed:
added thead and tbody elements

tupes commented on applications/issue-tracker/Home.html:
```diff
@@ -0,0 +1,92 @@
+
+    <body>
+        <nav>
+            <ul>
+                <header><h1><a href="Home.html">Issue Tracker</a></h1></header>
```
`<ul>` elements can only contain `<li>` elements

tupes reviewed (dismissed):
Good job! 

Just one thing to fix. Also, you should change the name of the file to `home.html` to follow naming conventions. Optional: you could put the date values inside of `<date>` elements.

gravinblumenn committed:
fixed the file name and date error

added <time datetime=""></time> for each of the date values and changed the file name back to home.html for consistency

JoshMacSween reviewed (approved):
Looks good to me, Gavin! 👍
